package.json

```diff
@@ -37,8 +37,7 @@
 		"lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. src/ && eslint --ignore-path .gitignore src/",
 		"format": "prettier --ignore-path .gitignore --write --plugin-search-dir=. src/",
 		"commit-preverify": "npx lint-staged && npm run check",
-		"commit": "npm run commit-preverify && cross-env NO_VERIFY=1 npx cz",
-		"release": "standard-version --no-verify"
+		"commit": "npm run commit-preverify && cross-env NO_VERIFY=1 npx cz"
```

Comment:
this should be..
pnpm run

Reply:
there is also pnpx, as a substitute to npx

Reply:
and if you agree @Tal500, I can add a few tags in package.json to for pnpm only.  let me know

Reply:
> this should be.. pnpm run

I can, I just want it to be resilient to a weak environment(e.g. automated scripting/github actions, etc.), in the case pnpm isn't installed.

Reply:
If you really want you may change them, just wanted to be "safe" or "portable"

Reply:
it's fine as is,
 I forgot about the actions, you are right.